askama/src/filters/humansize.rs

```diff
@@ -33,14 +34,18 @@ pub struct FilesizeFormatFilter(f32);
 impl fmt::Display for FilesizeFormatFilter {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        Ok(self.write_into(f)?)
+        Ok(self.write_into(f, NO_VALUES)?)
     }
 }
 
 impl FastWritable for FilesizeFormatFilter {
-    fn write_into<W: fmt::Write + ?Sized>(&self, dest: &mut W) -> crate::Result<()> {
+    fn write_into<W: fmt::Write + ?Sized>(
+        &self,
+        dest: &mut W,
+        _: &dyn Values,
```

Comment:
Well, we could name this argument and pass it to `write_into` instead of having `NO_VALUES` repeated all over, no? Not sure if there is a benefit for this approach compared to the one you took though...

Reply:
Yes, makes sense to simply pass the values along. The `write_into` implementations for u32 and &str don't use the value, but the optimizer knows that, too, and can kill the variable itself.